tethys_cli/install_commands.py

```diff
@@ -748,24 +757,35 @@ def install_command(args):
                     if has_module(conda_run):
                         conda_config = requirements_config["conda"]
                         install_packages(
-                            conda_config, update_installed=args.update_installed
+                            conda_config,
+                            update_installed=args.update_installed,
                         )
                     else:
-                        write_warning(
-                            "Conda is not installed. Attempting to install conda packages with pip..."
+                        write_warning("Conda is not installed...")
+                        proceed = input(
```

Comment:
Please check for `quiet is True` and don't prompt if it is to prevent interaction.